docs/distributions/gentoo/faq.md

```diff
@@ -0,0 +1,5 @@
+# Updating Kernel
```

Comment:
1. This section title should be in a question since this page is a faq page.
2. If there are no other questions for the faq, this page should be something different. Maybe a specific page just for updating.